src/main/assets/js/session-timeout.ts

```diff
@@ -12,12 +12,19 @@ const saveBeforeSessionTimeout = async () => {
     const formData = new FormData(form as HTMLFormElement);
     formData.append('saveBeforeSessionTimeout', 'true');
     const url = window.location.pathname;
-    const body = Object.fromEntries(formData);
+    const body = {};
+    for (const [key, value] of formData) {
+      if (body.hasOwnProperty(key)) {
+        body[key] = [...body[key], value];
+      } else {
+        body[key] = value;
+      }
+    }
     await fetch(url, {
       method: 'POST',
       headers: {
         'Content-type': 'application/json',
-        'csrf-token': body._csrf as string,
+        'csrf-token': body['_csrf'] as string,
```

Comment:
surely this is the same?

Reply:
its not needed anymore, but ide was complaining it might not exist.